my-react-app/src/components/Buttons/Buttons.tsx

```diff
@@ -13,11 +17,26 @@ const Buttons = () => {
     "bg-purple-400",
     "bg-teal-400",
   ];
+  const layoutClass = () => {
```

Comment:
dafür musst du keine funktion bauen dass kannst du direkt in die const rein geben mit simplen contionals:

`const layoutClass = layout === "grid" ? "flex flex-column flex-wrap" : "grid grid-cols-3";`